test/aqua/test_amplitude_estimation.py

```diff
@@ -104,7 +106,11 @@ def qasm(shots=100):
         [0.2, MaximumLikelihoodAmplitudeEstimation(2), {'estimation': 0.2}],
         [0.4, MaximumLikelihoodAmplitudeEstimation(4), {'estimation': 0.4}],
         [0.82, MaximumLikelihoodAmplitudeEstimation(5), {'estimation': 0.82}],
-        [0.49, MaximumLikelihoodAmplitudeEstimation(3), {'estimation': 0.49}]
+        [0.49, MaximumLikelihoodAmplitudeEstimation(3), {'estimation': 0.49}],
```

Comment:
I'm not jazzed about the fact that we only have e2e tests here and no unit tests, but we don't need to block on this because we need them for all the AE flavors. Note that we'll need to correct this later.

Reply:
It's on the list, we'll add extensive tests for all AE flavors soon.